src/headlessTask/index.js

```diff
@@ -73,11 +78,23 @@ async function callMatchingContext(savedUrlMatchingContext) {
   });
 }
 
+async function getHTMLOfCurrentChromeURL(eventMessageFromChromeURL) {
+  HTML = await fetch(`http://www.${eventMessageFromChromeURL}`).then(function (
+    response
+  ) {
+    // The API call was successful!
+    return response.text();
+  });
+}
+
 const HeadlessTask = async (taskData) => {
   SharedPreferences.getItem('url', async function (savedUrlMatchingContext) {
     callActionListeners();
     FloatingModule.initialize();
-    await callMatchingContext(savedUrlMatchingContext);
+    await Promise.all([
+      await callMatchingContext(savedUrlMatchingContext),
+      await getHTMLOfCurrentChromeURL(taskData.url),
```

Comment:
It seems that we don’t care about `getHTMLOfCurrentChromeURL` return value … is it useful ?

Reply:
I don't think I understand the question well. What do you mean? Example?

Reply:
I mean that `getHTMLOfCurrentChromeURL` returns a promise of `string` but here we call it but do not do anything with the response … it seems that `getHTMLOfCurrentChromeURL` is an old function not used anymore ?

Reply:
Ok this is solved